pkg/plugin/util/util.go

```diff
@@ -80,6 +80,23 @@ func InsertCode(filename, target, code string) error {
 	return os.WriteFile(filename, []byte(out), 0644)
 }
 
+// InsertCodeIfNotExist insert code if it does not already exists
+func InsertCodeIfNotExist(filename, target, code string) error {
```

Comment:
Why you need to create this one?
We scaffold the rules for all CRDs?
Why we need InsertCode if not exist>